src/components/Page.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/api/api';
+
+function Page() {
+  const { booksReducer } = useSelector((state) => state);
+  const { books } = booksReducer;
+  const [booksDisplay, setBooksDisplay] = useState(null);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (books !== booksDisplay) {
+      setBooksDisplay(books);
+    }
+  });
```

Comment:
- [ ] Server requests should be made in the componentDidMount lifecycle method or with react hooks. Here is supposed to have the `action` dispatched to the server to get the books. Kindly fix that